src/tests/IntegrationTests/Events/RepositoryManagerTests.cs

```diff
@@ -504,7 +505,67 @@ await RepositoryManager.SwitchBranch("branch2")
             repositoryManagerListener.DidNotReceive().OnLocksUpdated(Args.EnumerableGitLock);
         }
 
-        private void WaitForNotBusy(RepositoryManagerEvents managerEvents, int seconds = 1)
+		[Test]
+		public async Task ShouldCheckoutFiles()
+		{
+			await Initialize( TestRepoMasterCleanSynchronized );
+
+			var repositoryManagerListener = Substitute.For<IRepositoryManagerListener>();
+			repositoryManagerListener.AttachListener( RepositoryManager, repositoryManagerEvents );
+
+			var expected = new GitStatus
+			{
+				Behind = 1,
+				LocalBranch = "master",
+				RemoteBranch = "origin/master",
+				Entries =
+					new List<GitStatusEntry> {
+						new GitStatusEntry("foobar.txt", TestRepoMasterCleanSynchronized.Combine("foobar.txt"),
+							"foobar.txt", GitFileStatus.Untracked)
+					}
+			};
+
+			var result = new GitStatus();
+			Environment.Repository.OnStatusUpdated += status => { result = status; };
+
+			var foobarTxt = TestRepoMasterCleanSynchronized.Combine( "foobar.txt" );
+			foobarTxt.WriteAllText( "foobar" );
+
+			await TaskManager.Wait();
+			RepositoryManager.WaitForEvents();
+			WaitForNotBusy( repositoryManagerEvents, 1 );
+
+			repositoryManagerListener.Received().OnStatusUpdate( Args.GitStatus );
+			repositoryManagerListener.DidNotReceive().OnActiveBranchChanged( Arg.Any<ConfigBranch?>() );
+			repositoryManagerListener.DidNotReceive().OnActiveRemoteChanged( Arg.Any<ConfigRemote?>() );
+			repositoryManagerListener.DidNotReceive().OnLocalBranchListChanged();
+			repositoryManagerListener.DidNotReceive().OnRemoteBranchListChanged();
+			repositoryManagerListener.Received().OnIsBusyChanged( Args.Bool );
+			repositoryManagerListener.DidNotReceive().OnLocksUpdated( Args.EnumerableGitLock );
```

Comment:
This should be indented with spaces. There should be no spaces after `(` and before `)`